.github/dependabot.yml

```diff
@@ -10,6 +10,11 @@ updates:
     schedule:
       interval: daily
     open-pull-requests-limit: 10
+  - package-ecosystem: npm
+    directory: "/packages/node"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
```

Comment:
What happens when we hit this limit (i.e. will my command-line `git push origin feat/xyz` fail with a helpful error message)...? And isn't 10 a bit low given Dependabot can fire off loads of PR's in one go...?

Reply:
This is a cap to specifically limit the number of Dependabot PRs in parallel, so that would not impact the PRs we open manually.

Reply:
Ah ha - thanks (I didn't notice the name of the file itself, i.e. `dependabot.yml` !). Cheers.